.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Would it make sense to remove this file from VCS and instead add it to `.gitignore` to give folks the flexibility to change it how they wish?

Reply:
Yes, missed that.